packages/header-checker-lint/src/header-checker-lint.ts

```diff
@@ -74,6 +75,23 @@ class Configuration {
 
 export = (app: Application) => {
   app.on('pull_request', async context => {
+    let remoteConfiguration: ConfigurationOptions = DEFAULT_CONFIGURATION;
```

Comment:
Assuming `DEFAULT_CONFIGURATION` is of type `ConfigurationOptions`, the explicit type of `: ConfigurationOptions` is not needed.

Reply:
Done